examples/eight_schools/eight_schools.py

```diff
@@ -0,0 +1,94 @@
+"""implement the stan 8 schools example
+using the recommended non-centred parameterisation
+
+https://github.com/stan-dev/rstan/wiki/RStan-Getting-Started
```

Comment:
Can you move links to bottom of docstring?

Followed by References subsection, e.g.,
```
#### References

http://...
```